integration/test/dashboard_json_test.go

```diff
@@ -73,10 +71,12 @@ var restCounterMap = map[string]struct{}{
 // excludeCounters consists of counters which should be excluded from both Zapi/Rest in CI test
 var excludeCounters = []string{
 	"aggr_physical_",
-	"change_log",
 	"audit_log",
+	"change_log",
 	"cifs_session",
 	"cluster_peer",
+	// Based on the cluster upgrade, these metrics existence would be changed.
```

Comment:
if we sort this list, this comment does not make sense. It applied to five counters on the left side before. I think it's fine to remove